qa/python_models/python_based_backends/vllm_opt/model.json

```diff
@@ -0,0 +1,5 @@
+{
+    "model": "facebook/opt-125m",
+    "disable_log_requests": "true",
+    "gpu_memory_utilization": 0.25
+}
```

Comment:
can we copy this file and `config` from vllm_backend repo `samples` folder? This way we can eliminate the necessity to maintain the `vllm_opt` repo here

Reply:
Updated test.sh to use vllm_backend repo samples.